packages/embark-test-runner/package.json

```
@@ -52,7 +52,9 @@
     "embark-i18n": "^4.1.1",
     "embark-utils": "^4.1.1",
     "fs-extra": "7.0.1",
-    "istanbul": "0.4.5",
+    "istanbul-lib-coverage": "^2.0.5",
```

```suggestion
    "istanbul-lib-coverage": "2.0.5",
```

Should drop the `^` on lines 56 and 57 also. We can get `yarn.lock` churn if we don't use exact versions. I experienced that earlier today because of some deps in `embark-solc` that had caret ranges, which I changed to exact versions.

At some point I'd like to revisit, generally, the use of exact versions vs. version ranges in the various `package.json` of our monorepo. But for now our CI and release logic assumes that when running `yarn install` if `yarn.lock` changes as a result then the build/release will fail, i.e. all needed `yarn.lock` changes should already be committed.

---

Oh good point. I just thought we’d get bug fixes for free that way.